packages/generator/src/edmx-to-vdm/v4/entity.spec.ts

```diff
@@ -182,27 +182,34 @@ describe('entity', () => {
     });
 
     const entities = generateEntitiesV4(service, [], [], getFormatter());
+    const functionFilter = op => op.type === 'function';
+    const actionFilter = op => op.type === 'action';
 
     // entities[0]
-    expect(entities[0].functions.map(({ name }) => name)).toEqual([
-      'fn1IsBound'
-    ]);
-    expect(entities[0].functions[0].parameters.length).toBe(1);
-    expect(entities[0].actions.map(({ name }) => name)).toEqual([
-      'action1IsBound'
-    ]);
-    expect(entities[0].actions[0].parameters.length).toBe(0);
+    expect(
+      entities[0].operations.filter(functionFilter).map(({ name }) => name)
+    ).toEqual(['fn1IsBound']);
+    expect(
+      entities[0].operations.filter(functionFilter)[0].parameters.length
+    ).toBe(1);
+    expect(
+      entities[0].operations.filter(actionFilter).map(({ name }) => name)
+    ).toEqual(['action1IsBound']);
+    expect(
+      entities[0].operations.filter(actionFilter)[0].parameters.length
+    ).toBe(0);
```

Comment:
[pp] I think this would be easier to read as follows (the order might need to be adjusted, same for `entities[1]`):
```suggestion
    const [function, action] = entities[0].operations;
    
    // function
    expect(function.name).toEqual('fn1IsBound');
    expect(function.parameters.length).toBe(1);
    
    // action
    expect(action.name).toEqual('action1IsBound');
    expect(action.parameters.length).toBe(0);
```